### DISCUSSION TITLE
Improve/Change autoload of https://www.heroesprofile.com/Global/Matchups/Talents

### DISCUSSION THREAD

**Discussion by Zemill** [Heroes-Profile/heroesprofile#1028] [Ideas]
Change https://www.heroesprofile.com/Global/Matchups/Talents to allow for both heroes to be selected before filtering occurs.

Maybe another filter button

**Comment by fycalichking** [comment]
I suggest having the search only happen after clicking on the "filter" button.

**Comment by Br1ghtw1ng** [comment]
I agree

**Comment by Zemill** [comment]
On initial load of the page before any heroes are selected, the tool does not try and load any data.  If you have previously made some choices and then want to do a new comparison, you'd either have to reload the page to reset it, or it will load the first change and then the second change.

Two options

1. Change it to only work when you hit the filter button.
2. Have a reset button
